plugins/woocommerce-admin/client/marketplace/components/product-loader/product-loader.scss

```
@@ -1,6 +1,10 @@
 @import '../../stylesheets/_variables.scss';
 
 .woocommerce-marketplace {
+	&__product-loader {
+		margin-top: 16px;
```

I think you could use `$grid-unit-20` (from core base styles) or `$medium-gap` (our alias for the base style).

```suggestion
		margin-top: $grid-unit-20;
```

---

Ah, same with my previous message I don't know what I was thinking of when hardcoding these values in. Thank you for pointing them out.